src/webextension/widgets/button.css

```diff
@@ -36,6 +38,39 @@
   background-color: transparent;
 }
 
+.link-theme {
+  color: #0060df;
+  background-color: transparent;
+}
+
+.link-theme:global(.external)::after {
+  background-image: url(/icons/external-link.svg);
+  background-repeat: no-repeat;
+  background-size: 16px 16px;
+  content: "";
+  display: inline-block;
+  height: 16px;
+  margin: -.3rem .15rem 0 .25rem;
+  vertical-align: middle;
+  width: 16px;
+}
+
+.link-theme:active {
+  color: #003eaa;
+  text-decoration: underline;
+}
+
+.link-theme:hover {
+  color: #0060df;
+  text-decoration: underline;
+  background-color: transparent;
+}
+
+.link-theme:focus {
+  color: #0060df;
+  border-color: #0a84ff
+}
+
```

Comment:
All this should be removed, now that we have an `<ExternalLink/>` widget.

Reply:
>All this should be removed, now that we have an `<ExternalLink/>` widget.

I suppose this is all based on the above decisions re: "Sign in.." and "Copy.." being buttons with link theme vs buttons with.. button-y themes..

Point being: not all links are external which this is why we would keep this around (based on the current designs, some buttons look like links) right?

We had agreed the only places for true `<a>` links were as you see now, right (Feedback, FAQ)? Or have I misunderstood this feedback?